libs/shared/src/main/java/io/crate/common/collections/Sets.java

```diff
@@ -48,6 +49,14 @@ public static <T> boolean haveEmptyIntersection(Set<T> left, Set<T> right) {
         return left.stream().noneMatch(right::contains);
     }
 
+    public static <T> Set<T> append(Set<T> left, T ... right) {
```

Comment:
since the name is `append`, as a user of the method I would expect to return `left` with all the `right`s appended, and not a new set, but I also don't have a good suggestion for the name.

Reply:
In `Lists2` we call this `concat`

Reply:
concat is good!